agent/agent.go

```diff
@@ -1982,12 +2032,11 @@ func (a *Agent) loadServices(conf *Config) error {
 // unloadServices will deregister all services other than the 'consul' service
```

Comment:
This comment is stale (the `consul` service is no longer special).

Reply:
done